### PR TITLE
Add bydbctl and web-ui interacting guide.

### PR DESCRIPTION
**Pull request by wankai123**
- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Fixes apache/skywalking#<issue number>.
- [X] Update the [`CHANGES` log](https://github.com/apache/skywalking-banyandb/blob/main/CHANGES.md).

### CODE REVIEW

File: docs/interacting/bydbctl/bydbctl.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# bydbctl
+`bydbctl` is the command line tool for interacting with BanyanDB. It is a powerful tool that can be used to create, update, read, and delete schemas. It can also be used to query data stored in streams, measures, and properties.
+
+These are several ways to install:
+
+* Get binaries from [download](https://skywalking.apache.org/downloads/).
```
Comment (wu-sheng) [inline docs/interacting/bydbctl/bydbctl.md:6]: I can't find the ctl download link here.

Do we need a website update?

File: docs/interacting/bydbctl/bydbctl.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+* Build from [sources](https://github.com/apache/skywalking-banyandb/tree/main/bydbctl) to get latest features.
```
Comment (wu-sheng) [inline docs/interacting/bydbctl/bydbctl.md:7]: Source should be from website and svn/download path as well.

File: docs/interacting/bydbctl/bydbctl.md (hunk 1 above)
Comment (hanahmily) [reply docs/interacting/bydbctl/bydbctl.md:6]: They are in the same tar with the released server binary

image

We should add this detail here.

File: docs/interacting/bydbctl/query/measure.md [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+tagProjection:
+  tagFamilies:
+    - name: "storage-only"
+      tags: ["entity_id"]
+fieldProjection:
+  names: ["total", "value"]
+EOF
+```
+
+### Query with filter
```
Comment (hanahmily) [inline docs/interacting/bydbctl/query/measure.md:102]: We need more examples of `criteria`

1. logical expression: `and`, `or`
2. sub expression: like subquery of a SQL

 ```yaml
criteria:
  le:
    op: "LOGICAL_OP_OR"
    left:
      condition:
        name: "entity_id"
        op: "BINARY_OP_EQ"
        value:
          str:
            value: "entity_4"
    right:
      le:
        op: "LOGICAL_OP_OR"
        left:
          condition:
            name: "entity_id"
            op: "BINARY_OP_EQ"
            value:
              str:
                value: "entity_5"
```

4. more binary operations: 

-  Not equal: NE
-  Numbric: GT, GE, LT, LE
-  Having & In
-  Match: How to set the analyzer to the index rule.

5. Order by time or an index.

File: docs/interacting/bydbctl/query/measure.md [hunk 4]
```diff
@@ -0,0 +1,221 @@
+# Query [Measures](../../../concept/data-model.md#measures)
```
Comment (hanahmily) [inline docs/interacting/bydbctl/query/measure.md:1]: We need a page that describes how to set the topNAggregation and retrieve the result.

File: docs/interacting/web-ui.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
+# Web application
+
+The web application is hosted at [skywalking-banyandb-webapp](http://localhost:17913/) when you boot up the BanyanDB server.
+The web ui could use to query data stored in streams, measures, and properties.
+
```
Comment (hanahmily) [inline docs/interacting/web-ui.md:5]: The first section should be the dashboard. You could copy the [screenshot](https://github.com/apache/skywalking-banyandb/blob/main/docs/installation/cluster.md#self-observability-dashboard) here

File: docs/interacting/web-ui.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+## Stream
```
Comment (hanahmily) [inline docs/interacting/web-ui.md:6]: Add, edit, and delete a specific stream on this page. Please add relevant operations here.

File: docs/interacting/web-ui.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+The stream page shows the data stored in the stream. The menu on the left side shows the list of streams, index-rule, index-rule-binding and organized by their groups. 
+
+The user can select the stream to view the data, the data is displayed in a table format and can query/filter the data by the following:
+- Select the `Time Range` to be queried. 
+- Select the `Tag Families` to be queried.
+- Modify/Write the query config directly on the page. The config can refer to the [bydbctl query stream](./bydbctl/query/stream.md).
+
+## Measure
+![web-ui-measure.png](https://skywalking.apache.org/doc-graph/banyandb/v0.7.0/web-ui/web-ui-measure.png)
```
Comment (hanahmily) [inline docs/interacting/web-ui.md:17]: Like the `Stream,` schema-relevant operations are supported on this page.

File: docs/interacting/web-ui.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+The user can select the measure to view the data, the data is displayed in a table format and can query/filter the data by the following:
+- Select the `Time Range` to be queried.
+- Select the `Tag Families` to be queried.
+- Select the `Fields` to be queried.
+- Modify/Write the query config directly on the page. The config can refer to the [bydbctl query measure](./bydbctl/query/measure.md).
+
+## Property
+![web-ui-property.png](https://skywalking.apache.org/doc-graph/banyandb/v0.7.0/web-ui/web-ui-property.png)
+
+The property page shows the data stored in the property. The menu on the left side shows the list of properties, index-rule, index-rule-binding and organized by their groups.
```
Comment (hanahmily) [inline docs/interacting/web-ui.md:30]:
```suggestion
The property page shows the data stored in the property. The menu on the left side shows the list of properties and organized by their groups.
```

File: docs/interacting/web-ui.md (hunk 6 above)
Comment (hanahmily) [reply docs/interacting/web-ui.md:6]: You can also view and manage the index-rule and index-rule-binding in a stream group in this page.

File: docs/interacting/web-ui.md (hunk 7 above)
Comment (hanahmily) [reply docs/interacting/web-ui.md:17]: More than the index-rule and index-rule-binding, topn-aggregation could be managed in this page.

**Review by hanahmily — CHANGES_REQUESTED** [review]

File: docs/interacting/bydbctl/bydbctl.md (hunk 1 above)
Comment (wu-sheng) [reply docs/interacting/bydbctl/bydbctl.md:6]: I don't mean svn path. I mean we don't have download link directly from the website.

We should add it.

Comment (wu-sheng) [reply docs/interacting/bydbctl/bydbctl.md:6]: No one but ourselves knows they are released in one place.

@wankai123 Could you add bydctl download project at this page? And target to the correct version and links.

Comment (wu-sheng) [reply docs/interacting/bydbctl/bydbctl.md:6]: https://github.com/apache/skywalking/issues/12510 creates for @hanahmily 

This relies on new release process and scripts.

**Review by hanahmily — APPROVED** [review]

**Pull request merged**